pkg/azurefile/controllerserver.go

```diff
@@ -104,6 +106,11 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 	}
 
 	if acquired := d.volumeLocks.TryAcquire(volName); !acquired {
+		// logging the job status if it's volume cloning
+		if req.GetVolumeContentSource() != nil {
+			jobState, percent, err := getAzcopyJob(volName)
+			klog.V(2).Infof("azcopy job status: %s, copy percent: %s%, error: %v", jobState, percent, err)
```

Comment:
`getAzcopyJob` func also prints similar logs as L112

Reply:
Delete all info logs in `getAzcopyJob`, retain warning logs if there is error in func. Print info logs out of func `getAzcopyJob`